backend/src/exceptions/bad-credentials/bad-credentials.exception.ts

```diff
@@ -0,0 +1,11 @@
+import { CustomExceptionName } from 'guruhub-shared/common/enums/exceptions/custom-exception-name.enum';
+
+class BadCredentials extends Error {
```

Comment:
```suggestion
class UserError extends HttpError {
```
we need to name exceptions by service name. also, extends it from HttpError